### PR TITLE
First delivery the project - Web service Rest

### PR DESCRIPTION
**Pull request by FELIPEGRANADA**
It's included: 

- Source code Java
- pom file
- postman collection file
- open api file
- properties file

### CODE REVIEW

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2007-present the original author or authors.
```
Comment (dannycadavid87) [inline .mvn/wrapper/MavenWrapperDownloader.java]: @FELIPEGRANADA please remove this unnecessary file

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.2/apache-maven-3.8.2-bin.zip
```
Comment (dannycadavid87) [inline .mvn/wrapper/maven-wrapper.properties]: delete this file also.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            			<goals>
+                			<goal>stop</goal>
+            			</goals>
+        			</execution>
+    			</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.springdoc</groupId>
+				<artifactId>springdoc-openapi-maven-plugin</artifactId>
+				<version>0.2</version>
```
Comment (dannycadavid87) [inline pom.xml:68]: please be careful with the version of this plugin

**Review by dannycadavid87 — APPROVED** [review]

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (FELIPEGRANADA) [reply .mvn/wrapper/MavenWrapperDownloader.java]: File deleted

File: .mvn/wrapper/maven-wrapper.properties (hunk 2 above)
Comment (FELIPEGRANADA) [reply .mvn/wrapper/maven-wrapper.properties]: File deleted

File: pom.xml (hunk 3 above)
Comment (FELIPEGRANADA) [reply pom.xml:68]: I understand. I will be careful with next project.

**Pull request merged**